Adding health report to logstash integration

flexitrev opened the pull request:
## Proposed commit message

Added data and dashboards including Logstash Health report

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [X] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Author's Checklist

<!-- Recommended
Add a checklist of things that are required to be reviewed in order to have the PR approved
-->
- [ ] 

## How to test this PR locally

<!-- Recommended
Explain here how this PR will be tested by the reviewer: commands, dependencies, steps, etc.
-->

## Related issues

Closes https://github.com/elastic/integrations/issues/12465


## Screenshots

### Node Health
![Node Health](https://github.com/user-attachments/assets/79a6c9b2-abca-417d-8f98-29db9066182a)

### Pipeline Health
![Pipeline Health](https://github.com/user-attachments/assets/72c6aa54-4b27-4527-ae5e-1c06e9776957)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/20941)
* Commit: 60d0b72a23decbd4322bafb9051e6384b5e131b6

### Failed CI Steps
* [Check integrations logstash](https://buildkite.com/elastic/integrations/builds/20941#01949a7a-9c6d-43a2-b3b4-e2d29157eaf8)

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/20941","number":20941,"commit":"60d0b72a23decbd4322bafb9051e6384b5e131b6"}],"number":20941}
buildkite-pr-comment-->

andrewkroh commented on packages/logstash/changelog.yml:
```diff
@@ -1,4 +1,9 @@
+  changes:
+    - description: Adding data and dashboards from Logstash Health Report
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/12394
```
I think you want 12464.

```suggestion
      link: https://github.com/elastic/integrations/pull/12464
```

yaauie commented:
- On the "General Pipeline Information" section:
   - I would not expect `TERMINATED` pipelines to have load averages; the presence of the 0's and green color in those columns _visually_ tells me that everything is A-okay.
   - Is "Load" in this section short-hand for Worker Utilization? Although it's shorter, I'm wary of overloading the word "Load" to mean new things (from a dev-ops perspective, it already has a very specific meaning related to CPU instructions queued per CPU tick)
   - Can we color-code the whole row based on the "Status" column (and if so, do we have prior-art for color-sets that are less in-your-face than #00ff00 #999999 #00ffff #ff0000 )?
   - It _feels_ like this is a smash-up of our two initial probes, but what happens when we add more?
     - in our diagnostic tool, we can compare two windows, IIRC it is always "current" against a selectable baseline, which lets us reduce the number of columns.

strawgate commented:
> * I would not expect `TERMINATED` pipelines to have load averages; the presence of the 0's and green color in those columns _visually_ tells me that everything is A-okay.

@flexitrev With the above sections covering terminated pipelines, we could change the bottom section to be running pipelines and just filter out terminated pipelines.

> Can we color-code the whole row based on the "Status" column (and if so, do we have prior-art for color-sets that are less in-your-face than #00ff00 #999999 #ffff00 #ff0000 )?

I do not believe you can color-code the whole row based on a cell -- you can color code the cell based on its value, though. As for the colors themselves, the color sets are the EUI palettes that ship with Kibana and will follow any changes the kibana team makes to the values for the colors. In general I think we should stick with the built-in palettes for integrations we ship to customers.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21110)
* Commit: de7548dbae1263d30dfa7dbf5d8fb4cf86e939e6

### Failed CI Steps
* [Check integrations logstash](https://buildkite.com/elastic/integrations/builds/21110#0194adaf-fc95-4adc-ad27-51205638b9e3)

### History
* :broken_heart: [Build #21018](https://buildkite.com/elastic/integrations/builds/21018) failed 3af6f720413a4f017ed1ca403b79e310769de493
* :broken_heart: [Build #20941](https://buildkite.com/elastic/integrations/builds/20941) failed 60d0b72a23decbd4322bafb9051e6384b5e131b6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21110","number":21110,"commit":"de7548dbae1263d30dfa7dbf5d8fb4cf86e939e6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21018","number":21018,"commit":"3af6f720413a4f017ed1ca403b79e310769de493"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/20941","number":20941,"commit":"60d0b72a23decbd4322bafb9051e6384b5e131b6"}],"number":21110}
buildkite-pr-comment-->

flexitrev commented:
Added a filter to the Worker utilization by pipeline to exclude terminated pipelines. Should I exclude finished too?

Colors are expected to change with project borealis - If they don't automatically get picked up by us picking builtin colors, we'll update later. https://github.com/elastic/search-team/issues/8943

flexitrev commented:
![pipeline_health](https://github.com/user-attachments/assets/4ba491d7-91ce-4e6f-9cc6-c84bceb12910)
 Updated dashboard, simplified worker utilization by pipeline table, modified visualization names

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21115)
* Commit: fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d

### Failed CI Steps
* [Check integrations logstash](https://buildkite.com/elastic/integrations/builds/21115#0194adde-ea68-4e83-888e-62cbee8ab8af)

### History
* :broken_heart: [Build #21114](https://buildkite.com/elastic/integrations/builds/21114) failed ecad5ac8a13345cb55fecd767104ae7cfae794dc
* :broken_heart: [Build #21110](https://buildkite.com/elastic/integrations/builds/21110) failed de7548dbae1263d30dfa7dbf5d8fb4cf86e939e6
* :broken_heart: [Build #21018](https://buildkite.com/elastic/integrations/builds/21018) failed 3af6f720413a4f017ed1ca403b79e310769de493
* :broken_heart: [Build #20941](https://buildkite.com/elastic/integrations/builds/20941) failed 60d0b72a23decbd4322bafb9051e6384b5e131b6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21115","number":21115,"commit":"fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21114","number":21114,"commit":"ecad5ac8a13345cb55fecd767104ae7cfae794dc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21110","number":21110,"commit":"de7548dbae1263d30dfa7dbf5d8fb4cf86e939e6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21018","number":21018,"commit":"3af6f720413a4f017ed1ca403b79e310769de493"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/20941","number":20941,"commit":"60d0b72a23decbd4322bafb9051e6384b5e131b6"}],"number":21115}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21124)
* Commit: 51cd04550d80eeaeccfcaab454f802e21f656b8b

### Failed CI Steps
* [Check integrations logstash](https://buildkite.com/elastic/integrations/builds/21124#0194ae45-ff22-4921-8741-0c1ed4260524)

### History
* :broken_heart: [Build #21115](https://buildkite.com/elastic/integrations/builds/21115) failed fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d
* :broken_heart: [Build #21114](https://buildkite.com/elastic/integrations/builds/21114) failed ecad5ac8a13345cb55fecd767104ae7cfae794dc
* :broken_heart: [Build #21110](https://buildkite.com/elastic/integrations/builds/21110) failed de7548dbae1263d30dfa7dbf5d8fb4cf86e939e6
* :broken_heart: [Build #21018](https://buildkite.com/elastic/integrations/builds/21018) failed 3af6f720413a4f017ed1ca403b79e310769de493
* :broken_heart: [Build #20941](https://buildkite.com/elastic/integrations/builds/20941) failed 60d0b72a23decbd4322bafb9051e6384b5e131b6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21124","number":21124,"commit":"51cd04550d80eeaeccfcaab454f802e21f656b8b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21115","number":21115,"commit":"fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21114","number":21114,"commit":"ecad5ac8a13345cb55fecd767104ae7cfae794dc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21110","number":21110,"commit":"de7548dbae1263d30dfa7dbf5d8fb4cf86e939e6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21018","number":21018,"commit":"3af6f720413a4f017ed1ca403b79e310769de493"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/20941","number":20941,"commit":"60d0b72a23decbd4322bafb9051e6384b5e131b6"}],"number":21124}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21131)
* Commit: 4aa76f35b8642ca6325eca22da029b9cc7f33681

### Failed CI Steps
* [Check integrations logstash](https://buildkite.com/elastic/integrations/builds/21131#0194ae80-6264-459d-b9d3-cb2578fec475)

### History
* :broken_heart: [Build #21124](https://buildkite.com/elastic/integrations/builds/21124) failed 51cd04550d80eeaeccfcaab454f802e21f656b8b
* :broken_heart: [Build #21115](https://buildkite.com/elastic/integrations/builds/21115) failed fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d
* :broken_heart: [Build #21114](https://buildkite.com/elastic/integrations/builds/21114) failed ecad5ac8a13345cb55fecd767104ae7cfae794dc
* :broken_heart: [Build #21110](https://buildkite.com/elastic/integrations/builds/21110) failed de7548dbae1263d30dfa7dbf5d8fb4cf86e939e6
* :broken_heart: [Build #21018](https://buildkite.com/elastic/integrations/builds/21018) failed 3af6f720413a4f017ed1ca403b79e310769de493

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21131","number":21131,"commit":"4aa76f35b8642ca6325eca22da029b9cc7f33681"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21124","number":21124,"commit":"51cd04550d80eeaeccfcaab454f802e21f656b8b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21115","number":21115,"commit":"fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21114","number":21114,"commit":"ecad5ac8a13345cb55fecd767104ae7cfae794dc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21110","number":21110,"commit":"de7548dbae1263d30dfa7dbf5d8fb4cf86e939e6"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21018","number":21018,"commit":"3af6f720413a4f017ed1ca403b79e310769de493"}],"number":21131}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21138)
* Commit: 976839ff69fbf14a20db48ef8a297d323b554bdf

### Failed CI Steps
* [Check integrations logstash](https://buildkite.com/elastic/integrations/builds/21138#0194aea5-1858-413d-90f4-a0759ef8be92)

### History
* :broken_heart: [Build #21131](https://buildkite.com/elastic/integrations/builds/21131) failed 4aa76f35b8642ca6325eca22da029b9cc7f33681
* :broken_heart: [Build #21124](https://buildkite.com/elastic/integrations/builds/21124) failed 51cd04550d80eeaeccfcaab454f802e21f656b8b
* :broken_heart: [Build #21115](https://buildkite.com/elastic/integrations/builds/21115) failed fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d
* :broken_heart: [Build #21114](https://buildkite.com/elastic/integrations/builds/21114) failed ecad5ac8a13345cb55fecd767104ae7cfae794dc
* :broken_heart: [Build #21110](https://buildkite.com/elastic/integrations/builds/21110) failed de7548dbae1263d30dfa7dbf5d8fb4cf86e939e6

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21138","number":21138,"commit":"976839ff69fbf14a20db48ef8a297d323b554bdf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21131","number":21131,"commit":"4aa76f35b8642ca6325eca22da029b9cc7f33681"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21124","number":21124,"commit":"51cd04550d80eeaeccfcaab454f802e21f656b8b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21115","number":21115,"commit":"fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21114","number":21114,"commit":"ecad5ac8a13345cb55fecd767104ae7cfae794dc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21110","number":21110,"commit":"de7548dbae1263d30dfa7dbf5d8fb4cf86e939e6"}],"number":21138}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21141)
* Commit: b09b07c412f1b95c6dc9e4d519e89329af123485

### Failed CI Steps
* [Check integrations logstash](https://buildkite.com/elastic/integrations/builds/21141#0194aebe-bfba-4413-a1c2-a1214528917d)

### History
* :broken_heart: [Build #21138](https://buildkite.com/elastic/integrations/builds/21138) failed 976839ff69fbf14a20db48ef8a297d323b554bdf
* :broken_heart: [Build #21131](https://buildkite.com/elastic/integrations/builds/21131) failed 4aa76f35b8642ca6325eca22da029b9cc7f33681
* :broken_heart: [Build #21124](https://buildkite.com/elastic/integrations/builds/21124) failed 51cd04550d80eeaeccfcaab454f802e21f656b8b
* :broken_heart: [Build #21115](https://buildkite.com/elastic/integrations/builds/21115) failed fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d
* :broken_heart: [Build #21114](https://buildkite.com/elastic/integrations/builds/21114) failed ecad5ac8a13345cb55fecd767104ae7cfae794dc

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21141","number":21141,"commit":"b09b07c412f1b95c6dc9e4d519e89329af123485"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21138","number":21138,"commit":"976839ff69fbf14a20db48ef8a297d323b554bdf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21131","number":21131,"commit":"4aa76f35b8642ca6325eca22da029b9cc7f33681"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21124","number":21124,"commit":"51cd04550d80eeaeccfcaab454f802e21f656b8b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21115","number":21115,"commit":"fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21114","number":21114,"commit":"ecad5ac8a13345cb55fecd767104ae7cfae794dc"}],"number":21141}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21144)
* Commit: ad579c47c0b4d2a400e8f9ef9b615816fc264349

### Failed CI Steps
* [Check integrations logstash](https://buildkite.com/elastic/integrations/builds/21144#0194aecd-3f27-4ac2-822b-eeb8725cf95f)

### History
* :broken_heart: [Build #21141](https://buildkite.com/elastic/integrations/builds/21141) failed b09b07c412f1b95c6dc9e4d519e89329af123485
* :broken_heart: [Build #21138](https://buildkite.com/elastic/integrations/builds/21138) failed 976839ff69fbf14a20db48ef8a297d323b554bdf
* :broken_heart: [Build #21131](https://buildkite.com/elastic/integrations/builds/21131) failed 4aa76f35b8642ca6325eca22da029b9cc7f33681
* :broken_heart: [Build #21124](https://buildkite.com/elastic/integrations/builds/21124) failed 51cd04550d80eeaeccfcaab454f802e21f656b8b
* :broken_heart: [Build #21115](https://buildkite.com/elastic/integrations/builds/21115) failed fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d
* :broken_heart: [Build #21114](https://buildkite.com/elastic/integrations/builds/21114) failed ecad5ac8a13345cb55fecd767104ae7cfae794dc

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21144","number":21144,"commit":"ad579c47c0b4d2a400e8f9ef9b615816fc264349"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21141","number":21141,"commit":"b09b07c412f1b95c6dc9e4d519e89329af123485"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21138","number":21138,"commit":"976839ff69fbf14a20db48ef8a297d323b554bdf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21131","number":21131,"commit":"4aa76f35b8642ca6325eca22da029b9cc7f33681"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21124","number":21124,"commit":"51cd04550d80eeaeccfcaab454f802e21f656b8b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21115","number":21115,"commit":"fe14f0f007c6c9c3496194f5c42bcc90fbf2c08d"}],"number":21144}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21145)
* Commit: 6d05933f5b0b652608b32209adeabdce27fa5177

### Failed CI Steps
* [Check integrations logstash](https://buildkite.com/elastic/integrations/builds/21145#0194aee4-6d12-43ad-8f03-233889b509f1)

### History
* :broken_heart: [Build #21144](https://buildkite.com/elastic/integrations/builds/21144) failed ad579c47c0b4d2a400e8f9ef9b615816fc264349
* :broken_heart: [Build #21141](https://buildkite.com/elastic/integrations/builds/21141) failed b09b07c412f1b95c6dc9e4d519e89329af123485
* :broken_heart: [Build #21138](https://buildkite.com/elastic/integrations/builds/21138) failed 976839ff69fbf14a20db48ef8a297d323b554bdf
* :broken_heart: [Build #21131](https://buildkite.com/elastic/integrations/builds/21131) failed 4aa76f35b8642ca6325eca22da029b9cc7f33681
* :broken_heart: [Build #21124](https://buildkite.com/elastic/integrations/builds/21124) failed 51cd04550d80eeaeccfcaab454f802e21f656b8b

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21145","number":21145,"commit":"6d05933f5b0b652608b32209adeabdce27fa5177"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21144","number":21144,"commit":"ad579c47c0b4d2a400e8f9ef9b615816fc264349"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21141","number":21141,"commit":"b09b07c412f1b95c6dc9e4d519e89329af123485"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21138","number":21138,"commit":"976839ff69fbf14a20db48ef8a297d323b554bdf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21131","number":21131,"commit":"4aa76f35b8642ca6325eca22da029b9cc7f33681"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21124","number":21124,"commit":"51cd04550d80eeaeccfcaab454f802e21f656b8b"}],"number":21145}
buildkite-pr-comment-->

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21149)
* Commit: 13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b

### History
* :broken_heart: [Build #21145](https://buildkite.com/elastic/integrations/builds/21145) failed 6d05933f5b0b652608b32209adeabdce27fa5177
* :broken_heart: [Build #21144](https://buildkite.com/elastic/integrations/builds/21144) failed ad579c47c0b4d2a400e8f9ef9b615816fc264349
* :broken_heart: [Build #21141](https://buildkite.com/elastic/integrations/builds/21141) failed b09b07c412f1b95c6dc9e4d519e89329af123485
* :broken_heart: [Build #21138](https://buildkite.com/elastic/integrations/builds/21138) failed 976839ff69fbf14a20db48ef8a297d323b554bdf
* :broken_heart: [Build #21131](https://buildkite.com/elastic/integrations/builds/21131) failed 4aa76f35b8642ca6325eca22da029b9cc7f33681

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21149","number":21149,"commit":"13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21145","number":21145,"commit":"6d05933f5b0b652608b32209adeabdce27fa5177"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21144","number":21144,"commit":"ad579c47c0b4d2a400e8f9ef9b615816fc264349"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21141","number":21141,"commit":"b09b07c412f1b95c6dc9e4d519e89329af123485"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21138","number":21138,"commit":"976839ff69fbf14a20db48ef8a297d323b554bdf"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21131","number":21131,"commit":"4aa76f35b8642ca6325eca22da029b9cc7f33681"}],"number":21149}
buildkite-pr-comment-->

strawgate commented on packages/logstash/data_stream/health_report/agent/stream/stream.yml.hbs:
```diff
@@ -0,0 +1,64 @@
+  .as(resp, bytes(resp.Body)
+  .decode_json().as(body,
+    [   
+        [{"event.dataset": "logstash.health_report",
```
If we're not customizing the dataset to be specific to the document type then we don't need to populate it at all

Was having event.dataset set to node and pipeline causing an issue?

flexitrev replied:
Yes, that was a reason for failing CI

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21228)
* Commit: 9fcdccdb8da6e4874e615693e5eec7b5d923e33a

### History
* :green_heart: [Build #21149](https://buildkite.com/elastic/integrations/builds/21149) succeeded 13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b
* :broken_heart: [Build #21145](https://buildkite.com/elastic/integrations/builds/21145) failed 6d05933f5b0b652608b32209adeabdce27fa5177
* :broken_heart: [Build #21144](https://buildkite.com/elastic/integrations/builds/21144) failed ad579c47c0b4d2a400e8f9ef9b615816fc264349
* :broken_heart: [Build #21141](https://buildkite.com/elastic/integrations/builds/21141) failed b09b07c412f1b95c6dc9e4d519e89329af123485
* :broken_heart: [Build #21138](https://buildkite.com/elastic/integrations/builds/21138) failed 976839ff69fbf14a20db48ef8a297d323b554bdf
* :broken_heart: [Build #21131](https://buildkite.com/elastic/integrations/builds/21131) failed 4aa76f35b8642ca6325eca22da029b9cc7f33681

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21228","number":21228,"commit":"9fcdccdb8da6e4874e615693e5eec7b5d923e33a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21149","number":21149,"commit":"13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21145","number":21145,"commit":"6d05933f5b0b652608b32209adeabdce27fa5177"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21144","number":21144,"commit":"ad579c47c0b4d2a400e8f9ef9b615816fc264349"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21141","number":21141,"commit":"b09b07c412f1b95c6dc9e4d519e89329af123485"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21138","number":21138,"commit":"976839ff69fbf14a20db48ef8a297d323b554bdf"}],"number":21228}
buildkite-pr-comment-->

robbavey commented:
Some great and helpful stuff here!

Some comments:

### Add Logstash Integration

`Metrics (Elastic Agent)` _and_ `Metrics (Stack Monitoring)` are both enabled by default. We should only enable one by default, and warn that only one of these should be enabled.

<img width="574" alt="Screenshot 2025-01-29 at 4 00 37 PM" src="https://github.com/user-attachments/assets/10614db0-10df-4bea-9c5d-79aad706c669" />

--

`Metrics (Elastic Agent)` only has Health Report enabled by default - is this intentional? Plus some "technial preview" language still present.
<img width="581" alt="Screenshot 2025-01-29 at 4 03 04 PM" src="https://github.com/user-attachments/assets/253b0caf-c25b-4569-aa95-5c7c5c8e9dfa" />

## Node Health

Is it intentional for `Problem Nodes` to include details of all nodes, regardless of whether they are problematic? 
<img width="1584" alt="Screenshot 2025-01-29 at 4 04 13 PM" src="https://github.com/user-attachments/assets/ed3e451c-ac84-4334-b26d-ad3c1552e6a0" />

## Pipeline Health

Should we update the y-axis of `Pipeline Status over time`, to show something like "Percentage of Pipelines", rather than count, as the units are percentages?
Also, should we consider Dropping Partial Intervals on the Date Histogram to avoid showing potentially misleading partial results? 
<img width="1351" alt="Screenshot 2025-01-29 at 4 08 02 PM" src="https://github.com/user-attachments/assets/a6a759e2-f6aa-4c20-b8d3-588cbb7f6396" />

### Concerning Pipelines

Is it possible to include a "last reported state" and/or something like "last reported in this state" field:
  
eg: Last 15 minute view:
<img width="1582" alt="Screenshot 2025-01-29 at 4 11 01 PM" src="https://github.com/user-attachments/assets/c80466e7-acb6-4215-8a1c-8988646e3e7c" />

Last 30 minute view: 
<img width="1580" alt="Screenshot 2025-01-29 at 4 12 36 PM" src="https://github.com/user-attachments/assets/1764cc83-54f2-4549-a8cf-20b5d6904a06" />

Over a long enough time period, there is a decent chance of seeing a pipeline in a `loading` state, which may be misleading or alarming?

---

I'd also like to see drilldowns/links to the existing dashboards to help users track down where their bottlenecks could be

robbavey commented on packages/logstash/_dev/build/docs/README.md:
````diff
@@ -541,16 +541,113 @@ An example event for `node` looks as following:
+
+#### Example
+An example event for 'health_report' looks as following:
+```json
````
This should use the field/event macros as was worked on in https://github.com/elastic/integrations/pull/12461/files

flexitrev replied:
I think I've got it now

robbavey reviewed (changes requested):
Main comments left in body of PR

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21238)
* Commit: df631acd8a26d0e63bbde3b7a733b312bc21c35d

### History
* :green_heart: [Build #21228](https://buildkite.com/elastic/integrations/builds/21228) succeeded 9fcdccdb8da6e4874e615693e5eec7b5d923e33a
* :green_heart: [Build #21149](https://buildkite.com/elastic/integrations/builds/21149) succeeded 13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b
* :broken_heart: [Build #21145](https://buildkite.com/elastic/integrations/builds/21145) failed 6d05933f5b0b652608b32209adeabdce27fa5177
* :broken_heart: [Build #21144](https://buildkite.com/elastic/integrations/builds/21144) failed ad579c47c0b4d2a400e8f9ef9b615816fc264349
* :broken_heart: [Build #21141](https://buildkite.com/elastic/integrations/builds/21141) failed b09b07c412f1b95c6dc9e4d519e89329af123485
* :broken_heart: [Build #21138](https://buildkite.com/elastic/integrations/builds/21138) failed 976839ff69fbf14a20db48ef8a297d323b554bdf

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21238","number":21238,"commit":"df631acd8a26d0e63bbde3b7a733b312bc21c35d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21228","number":21228,"commit":"9fcdccdb8da6e4874e615693e5eec7b5d923e33a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21149","number":21149,"commit":"13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21145","number":21145,"commit":"6d05933f5b0b652608b32209adeabdce27fa5177"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21144","number":21144,"commit":"ad579c47c0b4d2a400e8f9ef9b615816fc264349"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21141","number":21141,"commit":"b09b07c412f1b95c6dc9e4d519e89329af123485"}],"number":21238}
buildkite-pr-comment-->

robbavey commented:
README.md makes sense now (should https://github.com/elastic/integrations/pull/12461 be closed or committed...) - but I'd like to see your thoughts on my [comments](https://github.com/elastic/integrations/pull/12464#issuecomment-2622878839) outside of code review

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21429)
* Commit: ea6b9f45a2e4348b77e40e0ed09d1f227fab191b

### Failed CI Steps
* [Check integrations logstash](https://buildkite.com/elastic/integrations/builds/21429#0194c745-15d1-4191-8fab-e9d5b1372a6f)

### History
* :green_heart: [Build #21238](https://buildkite.com/elastic/integrations/builds/21238) succeeded df631acd8a26d0e63bbde3b7a733b312bc21c35d
* :green_heart: [Build #21228](https://buildkite.com/elastic/integrations/builds/21228) succeeded 9fcdccdb8da6e4874e615693e5eec7b5d923e33a
* :green_heart: [Build #21149](https://buildkite.com/elastic/integrations/builds/21149) succeeded 13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b
* :broken_heart: [Build #21145](https://buildkite.com/elastic/integrations/builds/21145) failed 6d05933f5b0b652608b32209adeabdce27fa5177
* :broken_heart: [Build #21144](https://buildkite.com/elastic/integrations/builds/21144) failed ad579c47c0b4d2a400e8f9ef9b615816fc264349
* :broken_heart: [Build #21141](https://buildkite.com/elastic/integrations/builds/21141) failed b09b07c412f1b95c6dc9e4d519e89329af123485

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21429","number":21429,"commit":"ea6b9f45a2e4348b77e40e0ed09d1f227fab191b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21238","number":21238,"commit":"df631acd8a26d0e63bbde3b7a733b312bc21c35d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21228","number":21228,"commit":"9fcdccdb8da6e4874e615693e5eec7b5d923e33a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21149","number":21149,"commit":"13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21145","number":21145,"commit":"6d05933f5b0b652608b32209adeabdce27fa5177"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21144","number":21144,"commit":"ad579c47c0b4d2a400e8f9ef9b615816fc264349"}],"number":21429}
buildkite-pr-comment-->

flexitrev commented:
@robbavey I updated the defaults, and removed erroneous experimental release on Plugin and Pipeline dataset 
<img width="1300" alt="image" src="https://github.com/user-attachments/assets/2433dfe1-9ffc-49fd-bd61-5297bdcddf45" />

I don't know how to disable datasets by default and would like to address that in a subsequent dot version to make this report available. The previous version without the health report had all three datasets enabled by default. 

The filter for problem nodes is any node whose status is not "green", so a node not reporting status would show up.

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21431)
* Commit: 585343188f327a780afa3cee0b8ca543fce407c7

### History
* :broken_heart: [Build #21429](https://buildkite.com/elastic/integrations/builds/21429) failed ea6b9f45a2e4348b77e40e0ed09d1f227fab191b
* :green_heart: [Build #21238](https://buildkite.com/elastic/integrations/builds/21238) succeeded df631acd8a26d0e63bbde3b7a733b312bc21c35d
* :green_heart: [Build #21228](https://buildkite.com/elastic/integrations/builds/21228) succeeded 9fcdccdb8da6e4874e615693e5eec7b5d923e33a
* :green_heart: [Build #21149](https://buildkite.com/elastic/integrations/builds/21149) succeeded 13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b
* :broken_heart: [Build #21145](https://buildkite.com/elastic/integrations/builds/21145) failed 6d05933f5b0b652608b32209adeabdce27fa5177

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21431","number":21431,"commit":"585343188f327a780afa3cee0b8ca543fce407c7"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21429","number":21429,"commit":"ea6b9f45a2e4348b77e40e0ed09d1f227fab191b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21238","number":21238,"commit":"df631acd8a26d0e63bbde3b7a733b312bc21c35d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21228","number":21228,"commit":"9fcdccdb8da6e4874e615693e5eec7b5d923e33a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21149","number":21149,"commit":"13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21145","number":21145,"commit":"6d05933f5b0b652608b32209adeabdce27fa5177"}],"number":21431}
buildkite-pr-comment-->

robbavey commented:
To disable datasets by default, add `enabled: false` to the manifest.yml in all the `data_streams` for that dataset - eg, to disable the `stack monitoring` setting:

```patch
--- a/packages/logstash/data_stream/node/manifest.yml
+++ b/packages/logstash/data_stream/node/manifest.yml
@@ -8,6 +8,7 @@ elasticsearch:
 streams:
   - input: logstash/metrics
     title: Logstash node metrics
+    enabled: false
     description: Collect Logstash node metrics
     vars:
       - name: period
diff --git a/packages/logstash/data_stream/node_stats/manifest.yml b/packages/logstash/data_stream/node_stats/manifest.yml
index 450f8f13bd..a7504cb578 100644
--- a/packages/logstash/data_stream/node_stats/manifest.yml
+++ b/packages/logstash/data_stream/node_stats/manifest.yml
@@ -9,6 +9,7 @@ streams:
   - input: logstash/metrics
     title: Logstash node stats metrics
     description: Collect Logstash node stats metrics
+    enabled: false
     vars:
       - name: period
         type: text
```

> The filter for problem nodes is any node whose status is not "green", so a node not reporting status would show up.
Add this to the filter for `Problem Nodes` and it should go away
<img width="291" alt="Screenshot 2025-02-03 at 3 33 18 PM" src="https://github.com/user-attachments/assets/10b4d63c-34e7-41fd-a84b-40b8ff74f6c8" />

strawgate commented:
It looks like we removed event.dataset prematurely -- it looks like CI was failing because sample-event.json had two values for event.dataset: 
https://buildkite.com/elastic/integrations/builds/21145#0194aee4-6d12-43ad-8f03-233889b509f1

[build/test-coverage/coverage-logstash-pipeline-1738101594676982809-report.xml](https://buildkite.com/organizations/elastic/pipelines/integrations/builds/21145/jobs/0194aee4-6d12-43ad-8f03-233889b509f1/artifacts/0194aeed-d8dc-4be5-aa78-32b43bbdb5e0)

```
<testcase name="static test: Verify sample_event.json" classname="logstash.health_report" time="0.115375683">
<failure>test case failed: one or more errors found in document: [0] field "event.dataset" should have value in ["logstash.health_report"], it has "logstash.pipeline"</failure>
</testcase>
```

In the sample_event.json when CI was failing there were two `event.dataset` values in the document: https://github.com/elastic/integrations/commit/13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b

flexitrev commented:
Thanks! I tried it at the top level manifest but not on the individual streams - 
<img width="743" alt="image" src="https://github.com/user-attachments/assets/ae18113d-7bd2-4472-acdd-8c4579c35d70" />

flexitrev commented:
> It looks like we removed event.dataset prematurely -- it looks like CI was failing because sample-event.json had two values for event.dataset: https://buildkite.com/elastic/integrations/builds/21145#0194aee4-6d12-43ad-8f03-233889b509f1
> 
> [build/test-coverage/coverage-logstash-pipeline-1738101594676982809-report.xml](https://buildkite.com/organizations/elastic/pipelines/integrations/builds/21145/jobs/0194aee4-6d12-43ad-8f03-233889b509f1/artifacts/0194aeed-d8dc-4be5-aa78-32b43bbdb5e0)
> 
> ```
> <testcase name="static test: Verify sample_event.json" classname="logstash.health_report" time="0.115375683">
> <failure>test case failed: one or more errors found in document: [0] field "event.dataset" should have value in ["logstash.health_report"], it has "logstash.pipeline"</failure>
> </testcase>
> ```
> 
> In the sample_event.json when CI was failing there were two `event.dataset` values in the document: [13ed93d](https://github.com/elastic/integrations/commit/13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b)
> 
> ```
> "event": {
>   "agent_id_status": "verified",
>   "ingested": "2025-01-28T18:41:28Z",
>   "dataset": "logstash.health_report"
> },
> "event.dataset": "logstash.pipeline"
> }
> ```

@strawgate Yes, I thought it best to just keep logstash.health_report. Should I keep both somehow? Rename event.dataset:logstash.pipeline?

strawgate commented:
Ideally, there would be two sample events, the node sample event and the pipeline sample event?

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21504)
* Commit: a440b14f92bb07d4658df7f7037661432ebe61da

### History
* :green_heart: [Build #21431](https://buildkite.com/elastic/integrations/builds/21431) succeeded 585343188f327a780afa3cee0b8ca543fce407c7
* :broken_heart: [Build #21429](https://buildkite.com/elastic/integrations/builds/21429) failed ea6b9f45a2e4348b77e40e0ed09d1f227fab191b
* :green_heart: [Build #21238](https://buildkite.com/elastic/integrations/builds/21238) succeeded df631acd8a26d0e63bbde3b7a733b312bc21c35d
* :green_heart: [Build #21228](https://buildkite.com/elastic/integrations/builds/21228) succeeded 9fcdccdb8da6e4874e615693e5eec7b5d923e33a
* :green_heart: [Build #21149](https://buildkite.com/elastic/integrations/builds/21149) succeeded 13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b
* :broken_heart: [Build #21145](https://buildkite.com/elastic/integrations/builds/21145) failed 6d05933f5b0b652608b32209adeabdce27fa5177

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21504","number":21504,"commit":"a440b14f92bb07d4658df7f7037661432ebe61da"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21431","number":21431,"commit":"585343188f327a780afa3cee0b8ca543fce407c7"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21429","number":21429,"commit":"ea6b9f45a2e4348b77e40e0ed09d1f227fab191b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21238","number":21238,"commit":"df631acd8a26d0e63bbde3b7a733b312bc21c35d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21228","number":21228,"commit":"9fcdccdb8da6e4874e615693e5eec7b5d923e33a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21149","number":21149,"commit":"13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b"}],"number":21504}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/integrations/builds/21517)
* Commit: e5f3045abd614aa5cb1ad2f32e4a7e2e6cb43024

### Failed CI Steps
* [:pipeline::arrow_up: Upload Pipeline: .buildkite/pipeline.yml](https://buildkite.com/elastic/integrations/builds/21517#0194cdf9-7da7-4f1c-99bd-be6a9ce78a1d)

### History
* :green_heart: [Build #21504](https://buildkite.com/elastic/integrations/builds/21504) succeeded a440b14f92bb07d4658df7f7037661432ebe61da
* :green_heart: [Build #21431](https://buildkite.com/elastic/integrations/builds/21431) succeeded 585343188f327a780afa3cee0b8ca543fce407c7
* :broken_heart: [Build #21429](https://buildkite.com/elastic/integrations/builds/21429) failed ea6b9f45a2e4348b77e40e0ed09d1f227fab191b
* :green_heart: [Build #21238](https://buildkite.com/elastic/integrations/builds/21238) succeeded df631acd8a26d0e63bbde3b7a733b312bc21c35d
* :green_heart: [Build #21228](https://buildkite.com/elastic/integrations/builds/21228) succeeded 9fcdccdb8da6e4874e615693e5eec7b5d923e33a
* :green_heart: [Build #21149](https://buildkite.com/elastic/integrations/builds/21149) succeeded 13ed93d495d7cfdf6cc2ec0f46e6cda999f50d6b

<!--buildkite-pr-comment-integrations
{"builds":[{"buildStatus":{"state":"running","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21517","number":21517,"commit":"e5f3045abd614aa5cb1ad2f32e4a7e2e6cb43024"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21504","number":21504,"commit":"a440b14f92bb07d4658df7f7037661432ebe61da"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21431","number":21431,"commit":"585343188f327a780afa3cee0b8ca543fce407c7"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21429","number":21429,"commit":"ea6b9f45a2e4348b77e40e0ed09d1f227fab191b"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21238","number":21238,"commit":"df631acd8a26d0e63bbde3b7a733b312bc21c35d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/integrations/builds/21228","number":21228,"commit":"9fcdccdb8da6e4874e615693e5eec7b5d923e33a"}],"number":21517}
buildkite-pr-comment-->

elasticmachine commented:
Pinging @elastic/security-service-integrations (Team:Security-Service Integrations)

flexitrev commented:
Cutting a new branch and creating a new PR - https://github.com/elastic/integrations/pull/12601
